src/brpc/server.cpp

```diff
@@ -902,6 +902,17 @@ int Server::StartInternal(const butil::EndPoint& endpoint,
         _tl_options = ThreadLocalOptions();
     }
 
+    auto original_bthread_tag = _options.bthread_tag;
```

Comment:
这种校验性质的方法是否可以再提前执行？因为在此之前我看到有些堆上内存被申请，我不确定 server 启动失败业务侧的处理逻辑是什么？如果不是 fast fail 那么看起来可能存在潜在内存泄漏。
Can this verification method be executed earlier? Because I saw some heap memory being requested before this, I am not sure what the business logic is for server startup failure? If it is not a fast fail, then it seems that there may be a potential memory leak.